app/views/brexit_checker/_results_business_actions.html.erb

```diff
@@ -16,7 +16,7 @@
           <%= render 'results_criteria', criteria: business_results[:criteria] %>
           <div class="govuk-grid-column-two-thirds">
             <%= render "action_list", {
-              analytics_group: "#{t('brexit_checker.results.audiences.business.heading')}.",
+              analytics_group: "#{t('brexit_checker.results.audiences.business.heading')} - 1.",
```

Comment:
This fixes a bug i introduced when refactoring groups. I'll squash this back into the correct commit when folks are happy for this to me merged into the new-results-page branch.

Put some details about the output in the commit message [here](https://github.com/alphagov/finder-frontend/pull/1812/commits/1f9a8179c1539009b749c690901f43211a2b1225).